src/platform_impl/linux/window.rs

```diff
@@ -184,6 +184,7 @@ impl Window {
     if attributes.maximized {
       window.maximize();
     }
+    window.set_resizable(attributes.resizable);
```

Comment:
`window.set_resizable` is already called on Line 81, how does adding this fixes the issue?

Reply:
oh I forgot to remove this line...

Reply:
I removed it.
https://github.com/tauri-apps/tao/pull/456/commits/b726e310ab3c4c991010911b3d0b0335def03e03

Reply:
do you know if this fixes #302

Reply:
I don't know but I will test it

Reply:
I tested https://github.com/tauri-apps/tao/issues/302 in this PR, but this behavior is not occurred in initial build.
So I think this is occurred only in runtime.